### PR TITLE
Update:  keycloak integration with device-manager, cert-sidecar, gui-…

### PR DESCRIPTION
**Pull request by rodrigoaasm**
…nx and backstage.

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
* dojot/dojot#2408

### CODE REVIEW

**Review by gabrielifg — DISMISSED** [review]
Great Job!!!!

**Review by gabrielifg — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -14,7 +14,7 @@ DOJOT_VERSION=feature/keycloak-v2
 KEYCLOAK_MASTER_USER=master
 # set a password for the user master in KEYCLOAK_MASTER_PASSWORD, can't be empty
 # example KEYCLOAK_MASTER_PASSWORD=Master@2021
-KEYCLOAK_MASTER_PASSWORD=
+KEYCLOAK_MASTER_PASSWORD=master
```
[the remark this answers is not in the document]
Comment (rodrigoaasm) [reply .env:17]: Essa será aquela senha padrão da dojot né. No primeiro acesso o usuário vai ser obrigado a alterar a senha.

File: kong/kong.config.sh [hunk 2]
```diff
@@ -110,7 +110,7 @@ curl  -sS -X PUT \
 
 # service: gui
 
-createEndpoint "gui" "http://gui:80"  '"/"' "false"
+# createEndpoint "gui" "http://gui:80"  '"/"' "false"
```
[the remark this answers is not in the document]
Comment (rodrigoaasm) [reply kong/kong.config.sh:113]: Então essa ainda não é uma versão definitiva. Mas na versão definitiva será removido. Obrigado.

**Review by gabrielifg — APPROVED** [review]
GG!!

**Review by marianoleonardo — APPROVED** [review]

**Pull request merged**